tests/hacker.test.js

```diff
@@ -630,6 +702,30 @@ describe("PATCH update one hacker", function () {
         });
     });
 
+    // should fail to change hacker data with an unconfirmed email
+    it("should FAIL and FAIL to update the user's hacker info due to unconfirmed email", function (done) {
```

Comment:
What does "Fail and Fail mean"?

Reply:
changed to should FAIL and not update the user's hacker info...